### DISCUSSION TITLE
Setting up a Relay confuses me

### DISCUSSION THREAD

**Discussion by un-fuf-a-doo** [mastodon/mastodon#17187] [General]
We have two Mastodon hosts up, running, but not public yet.  We would like to set up our two instances as relays before going public. After adding my hosts, I see the status on both ends says "Waiting for Approval".  Where exactly is this approval sent? How is the approval completed?

Jeff

**Comment by ClearlyClaire** [comment]
Mastodon instances cannot currently operate as relays. They can subscribe to relays, which are services provided by other software.

**Reply by un-fuf-a-doo** [reply]
Understood. So there is no way to link our two servers.

**Reply by ClearlyClaire** [reply]
Not without the help of another piece of software, no. I am not sure which relay software is maintained nowadays, but you may give https://github.com/noellabo/pub-relay a try

**Reply by PhilipMay** [reply]
Are there other software relay options?

**Comment by wiegelmann** [comment]
@PhilipMay I'm using this one, runs quite stable and fast: https://github.com/yukimochi/Activity-Relay
